pkg/api/v1beta3/dynakube/dynakube_props.go

```diff
@@ -104,3 +104,7 @@ func (dk *DynaKube) ApiRequestThreshold() time.Duration {
 func (dk *DynaKube) IsTokenScopeVerificationAllowed(timeProvider *timeprovider.Provider) bool {
 	return timeProvider.IsOutdated(&dk.Status.DynatraceApi.LastTokenScopeRequest, dk.ApiRequestThreshold())
 }
+
+func (dk *DynaKube) FeatureBootstrapperInjection() bool {
+	return dk.FeatureDownloadViaJob() && !dk.OneAgent().IsCSIAvailable()
```

Comment:
I think it would make sense to rename the `FeatureDownloadViaJob` feature flag as the actual value is different and it's used not only for downloading via jobs now.

Reply:
https://github.com/Dynatrace/dynatrace-operator/pull/4515/commits/64d01a1750f1ad37cabdcb12984bdefa2e7d9013 done